.zuul.d/jobs.yaml

```diff
@@ -133,6 +133,51 @@
 
 # =============================================================================
 
+- job:
+    name: ansible-runner-build-container-image-stable-2.11
+    parent: ansible-build-container-image
+    provides:
+      - ansible-runner-stable-2.11-container-image
+    requires:
+      - ansible-core-container-image
```

Comment:
This really should be ansible-core-stable-2.11-container-image, which would come from:

https://github.com/ansible/ansible-core-image/blob/main/.zuul.d/jobs.yaml

However we haven't created a stable job before. If you want to try and hack on it great, I can likely build it maybe this evening

Reply:
i really not sure how to add that job. and what is the need for it? as in this job we are creating ansible-runner-stable-2.11-container-image image right so whats diffrance between ansible-runner-stable-2.11-container-image ,ansible-core-stable-2.11-container-image, ansible-core-container-image

Reply:
once we branch ansible/ansible for stable-2.11, the ansible-core-container-image will become 2.12 code base. So we'll need a 2nd new image.

That said, don't worry about it for now. I am going to refactor the job to remove the ansible-core image, as we don't use it downstream.